### PR TITLE
Add `Model` assessor to `ViewResult` for MVC5 portability.

### PR DESCRIPTION
**Pull request by NTaylorMullen**
#3495

### CODE REVIEW

File: src/Microsoft.AspNet.Mvc.ViewFeatures/ViewResult.cs [hunk 1]
```diff
@@ -29,6 +29,11 @@ public class ViewResult : ActionResult
         public string ViewName { get; set; }
 
         /// <summary>
+        /// Gets the view data model.
+        /// </summary>
+        public object Model => ViewData.Model;
```
Comment (Eilon) [inline src/Microsoft.AspNet.Mvc.ViewFeatures/ViewResult.cs]: Do we want `ViewData?.Model`?

Comment (NTaylorMullen) [inline src/Microsoft.AspNet.Mvc.ViewFeatures/ViewResult.cs]: Yup!

**Comment by pranavkm** [comment]
:shipit:

**Comment by NTaylorMullen** [comment]
:up: :date:

**Comment by Eilon** [comment]
:shipit:

**Pull request merged**